index.html

```diff
@@ -49,18 +49,21 @@
   </head>
 
   <body>
-    <h1 id="header">Basics: Scissors Paper Stone 🚀</h1>
+    <h1 id="header">Scissors Paper Stone - Original/ Reversed/ Korean! 🚀</h1>
+
     <div id="container">
-      <p>Input:</p>
+      <p>Hi! Please enter your user name first, then you may play away!</p>
```

Comment:
great initiative editing the html code such that it improves the player experience